public/js/discover/trie.js

```diff
@@ -0,0 +1,121 @@
+const categories = [
+  'canned food',
+  'produce',
+  'cleaning supplies',
+  'blankets',
+  'clothes',
+  'pet food',
+  'plastic bags',
+  'books',
+  'school supplies',
+  "children's toys",
+  'hair products',
+  'deoderant',
+  'soap',
+  'bottled drinks',
+  'computers',
+  'cars',
+];
```

Comment:
Question on this hardcoded list. Are we planning to switch to read this list from Firestore in post-MVP/MVP?

Reply:
Yep, we're aiming to read categories from the database for MVP 👍